velox/functions/lib/Repeat.cpp

```diff
@@ -184,9 +203,16 @@ static std::vector<std::shared_ptr<exec::FunctionSignature>> signatures() {
 }
 } // namespace
 
+// Disallows negative count.
 VELOX_DECLARE_VECTOR_FUNCTION(
     udf_repeat,
     signatures(),
-    std::make_unique<RepeatFunction>());
+    std::make_unique<RepeatFunction>(false));
+
+// Allows negative count.
+VELOX_DECLARE_VECTOR_FUNCTION(
```

Comment:
Let's remove these macros and instead add .h file with the following APIs:

```
std::shared_ptr<exec::VectorFunction> makeRepeat(bool allowNegativeCount);

std::vector<std::shared_ptr<exec::FunctionSignature>> repeatSignatures();
```

See velox/functions/lib/Re2Functions.h